### PR TITLE
Temenos Aerns

### PR DESCRIPTION
**Pull request by wrenffxi**
**_I affirm:_**
- [x] that I agree to Project Topaz's [Limited Contributor License Agreement](http://project-topaz.com/blob/release/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [x] that I've _tested my code_ since the last commit in the PR, and will test after any later commits

Assigned new jobs to Temenos Aerns, and adjusted their pools to match retail.

Thanks to Nyu for retail captures and determining which job each Aerns is on retail.

Spreadsheet we're using to fix Aerns is [here](https://docs.google.com/spreadsheets/d/1miQYoxtuk20sjWEKB-u97q1Fcvg6l55fjN_LQgnjHTQ/edit#gid=0).

### CODE REVIEW

**Review by zach2good — APPROVED** [review]
Looks sane to me, AND it won't cause conflicts with anyone else's work. Thats a win-win 🎉

File: sql/mob_pets.sql [hunk 1]
```diff
@@ -1135,18 +1135,21 @@ INSERT INTO `mob_pets` VALUES (16928827, 1, 15); -- Tonberry_s_Elemental
...
--- INSERT INTO `mob_pets` VALUES (16929102, 1, 9); -- Aern_s_Wynav
--- INSERT INTO `mob_pets` VALUES (16929114, 1, 9); -- Aern_s_Elemental
--- INSERT INTO `mob_pets` VALUES (16929115, 1, 9); -- Aern_s_Avatar
+
+-- ------------------------------------------------------------
+-- Temenos Aerns (Zone 37)
+-- ------------------------------------------------------------
+
+INSERT INTO `mob_pets` VALUES (16929055, 1, 14); -- Temenos Aern DRG
+INSERT INTO `mob_pets` VALUES (16929058, 1, 9); -- Temenos Aern BST
```
Comment (59blargedy) [inline sql/mob_pets.sql:1144]: my records show this 58 is a rdm, and 60 is the bst, after 2 runs. 
https://github.com/project-topaz/topaz/issues/513#issuecomment-652149187

File: sql/mob_spawn_points.sql [hunk 2]
```diff
@@ -8031,8 +8031,8 @@ INSERT INTO `mob_spawn_points` VALUES (16929059,'Aerns_Euvhi','Aern\'s Euvhi',97
 INSERT INTO `mob_spawn_points` VALUES (16929060,'Temenos_Aern','Temenos Aern',119,568.000,-0.500,-500.000,0);
```
Comment (59blargedy) [inline sql/mob_spawn_points.sql:8031]: this was the bst on my runs, with 16929058  being the rdm

File: sql/mob_spawn_points.sql [hunk 3]
```diff
@@ -8057,30 +8057,30 @@ INSERT INTO `mob_spawn_points` VALUES (16929085,'Temenos_Aern','Temenos Aern',12
 INSERT INTO `mob_spawn_points` VALUES (16929086,'Temenos_Aern','Temenos Aern',108,560.000,4.500,-404.000,64);
```
Comment (59blargedy) [inline sql/mob_spawn_points.sql:8057]: i had 16929082 as another rdm on my runs

File: sql/mob_pets.sql (hunk 1 above)
Comment (wrenffxi) [reply sql/mob_pets.sql:1144]: Hm.  I see that your list in issue 513 says that 16929058 is a RDM, but

1. The NPClogger data attached to 513 agrees with Nyu's cap that 16929058 has model 00005905, which is the axe model used by WAR/RNG/BST; and
2. The Aern's Euvhi with id 16929059 immediately follows this Aern, and can't belong to any other.  Pets always come after their master in the extracts (they don't always immediately follow, but they always come after; never before)

Similarly, 16929060 has model 00005A05, in both Nyu's cap and yours, which is the sword model used by RDM/PLD/DRK.  And no pet follows it in the data.

So I'm inclined to believe the jobs currently in this PR are correct.

File: sql/mob_spawn_points.sql (hunk 3 above)
Comment (wrenffxi) [reply sql/mob_spawn_points.sql:8057]: 16929082 is a RDM in this PR too.  (It already was one; this PR didn't change it.)

File: sql/mob_pets.sql (hunk 1 above)
Comment (59blargedy) [reply sql/mob_pets.sql:1144]: fair enough, data doesnt lie. thanks for taking a second look!

File: sql/mob_spawn_points.sql (hunk 3 above)
Comment (59blargedy) [reply sql/mob_spawn_points.sql:8057]: i did not see it here and still don't, but i'm sure i'm just missing something - was looking where it jumps from 16929081 to 16929086. not a comment on the pr, but the seemingly missing data. where is that spawn??? ill look around.

Comment (wrenffxi) [reply sql/mob_spawn_points.sql:8057]: I believe this is just a side effect of viewing the diff of a large file on github.  It's only showing you the edited lines plus a few on either side, and leaving out many of the unchanged lines between the edits.

**Pull request merged**